### PR TITLE
Add bearer user handler

### PR DESCRIPTION
**Pull request by richtier**
[this task](https://uktrade.atlassian.net/browse/ED-1799)

API will call `get_bearer_user` in the bearer token authenticator

### CODE REVIEW

File: directory_sso_api_client/user.py [hunk 1]
```diff
@@ -5,6 +5,7 @@ class UserAPIClient(BaseAPIClient):
 
     endpoints = {
         'session_user': 'session-user/',
+        'bearer_user': 'oauth2/user-profile/v1/',
```
Comment (konradko) [inline directory_sso_api_client/user.py]: `bearer token` is created is used by the oauth2 consumer. Endpoint name should be sth like `oauth2_user_profile`

File: directory_sso_api_client/user.py [hunk 2]
```diff
@@ -14,6 +15,13 @@ def get_session_user(self, session_id):
             params={'session_key': session_id}
         )
 
+    def get_bearer_user(self, bearer_token):
```
Comment (konradko) [inline directory_sso_api_client/user.py]: ditto `get_oauth2_user_profile`

File: directory_sso_api_client/user.py [hunk 3]
```diff
@@ -4,8 +4,9 @@
 class UserAPIClient(BaseAPIClient):
 
     endpoints = {
-        'session_user': 'session-user/',
-        'last_login': 'last-login/',
+        'session_user': 'api/v1/session-user/',
```
Comment (konradko) [inline directory_sso_api_client/user.py]: please change the SSO client base env var in each app that uses that in dev env + make a PR for stage / prod conf

Comment (richtier) [inline directory_sso_api_client/user.py]: sure, but I'll do the env var update in api once my API is using this release. If I make the change now, API will be more broken :)

**Pull request merged**